feature-libs/product-configurator/rulebased/components/product-title/configurator-product-title.component.spec.ts

```diff
@@ -237,21 +251,31 @@ describe('ConfigProductTitleComponent', () => {
   });
 
   describe('product$', () => {
-    it('should get product name as part of product of configuration', () => {
+    it('should get product name as part of product configuration', () => {
       component.product$.subscribe((data: Product | undefined) => {
         expect(data?.name).toEqual(PRODUCT_NAME);
+        expect(data?.code).toEqual(configuration.owner.id);
       });
     });
 
     it('should get product name as part of product from overview, in case configuration is order bound', () => {
-      configuration = orderEntryconfig;
+      configuration = orderEntryConfig;
+      component.product$.subscribe((data: Product | undefined) => {
+        expect(data?.name).toEqual(PRODUCT_NAME);
+        expect(data?.code).toEqual(configuration.owner.id);
```

Comment:
```suggestion
        expect(data?.code).toEqual(PRODUCT_CODE);
```
don't you think its more readable if we directly use the constant, instead of reading a property which gets initialized somewhere else?
applies also to at least one other expect below

Reply:
@Uli-Tiger, as you mentioned above the same product will be returned.